_locales/en/messages.json

```diff
@@ -1,55 +1,55 @@
 {
   "appDesc": {
     "message": "A text editor for Chrome OS and Chrome.",
-    "description":"The description of the application, displayed in the web store."
+    "description": "The description of the application, displayed in the web store."
```

Comment:
optional: ideally these formatting consistency changes should be in a separate patch, so that the bmale history makes more sense

Reply:
I'm actually so lost because these were already fixed in a previous CL

Reply:
Turns out i needed a rebase :)